doc/api/console.md

```diff
@@ -167,6 +167,52 @@ console.assert(false, 'this message will print, but no error thrown');
 console.log('this will also print');
 ```
 
+### console.clear()
+<!-- YAML
+added: REPLACEME
+-->
+
+When `stdout` is a TTY, calling `console.clear()` will attempt to clear the
+TTY. When `stdout` is not a TTY, this method does nothing.
+
+*Note*: The specific operation of `console.clear()` can vary across operating
+systems and terminal types. For most Linux operating systems, `console.clear()`
+operates similarly to the `clear` shell command. On Windows, `console.clear()`
+will clear only the output in the current terminal viewport for the Node.js
+binary.
+
+### console.count([label])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `label` {any} Any value (other than `Symbol`) that can be coerced into a
+  string.
```

Comment:
Would it be clearer to just say `label` should be a string? Sure, the function will *work* if it's passed an object, but the output `[object Object]: 1` is usually not what a user would want. Additionally it might be confusing that two distinct objects share a label (since they will both be coerced to `[object Object]`).